actionlist/package.json

```diff
@@ -34,4 +35,4 @@
   "jest": {
     "preset": "react-native"
   }
-}
+}
```

Comment:
Insert a newline at the end of the file (`npm` commands should do this for you if you use, e.g., `npm install --save-dev react-native-side-menu`, I'm not sure why it didn't).